README.md

```diff
@@ -97,142 +97,153 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+ - Added support to the Finnish language character set. Thanks [@davidtlascelles](https://github.com/davidtlascelles)!
+ - Added currency symbols for Croatian, Gaeilge, Gàidhlig and Welsh. Thanks [@PesBandi](https://github.com/PesBandi)!
+ - Added a missing Latin letter ꝡ. Thanks [@cubedhuang](https://github.com/cubedhuang)!
+ - Added fraction characters. Thanks [@PesBandi](https://github.com/PesBandi)!
+ - Added support to the Danish language character set. Thanks [@PesBandi](https://github.com/PesBandi)!
+ - Added the Kazakhstani Tenge character to the Currencies characters set. Thanks [@PesBandi](https://github.com/PesBandi)!
+ - Renamed Slovakian to Slovak, which is the correct term. Thanks [@PesBandi](https://github.com/PesBandi)!
+ - Added the Greek language character set. Thanks [@mcbabo](https://github.com/mcbabo)!
 
 ### Settings
 
- - Added a new Dashboard home page, with quick access for enabling modules, short descriptions and activation methods. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!
- - Fixed a typo in the Hosts File Editor page. Thanks [@Deepak-Sangle](https://github.com/Deepak-Sangle)!
- - Added a lock icon to the flyout listing of all modules when its enabled state is controlled by policy.
- - The "All apps" list in the flyout will now list all apps even if their enabled state is controlled by policy.
+ - When clicking a module's name on the Dashboard, it will navigate to that module's page.
+ - Fixed the clipping of information in the Backup and Restore section of the General Settings page. Thanks [@niels9001](https://github.com/niels9001)!
+ - Updated the File Explorer Add-ons fluent icon. Thanks [@niels9001](https://github.com/niels9001)!
+ - Added a warning when trying to set a shortcut that might conflict with "Alt Gr" key combinations.
+ - Added a direct link to the OOBE's "What's New page" from the main Settings window. Thanks [@iakrayna](https://github.com/iakrayna)!
+ - Changed mentions from Microsoft Docs to Microsoft Learn.
+ - Fixed the slow reaction to system theme changes.
 
-### Video Conference Mute
+### Text Extractor
 
- - Added an option to allow for the toolbar to hide after some time passed. Thanks [@quyenvsp](https://github.com/quyenvsp)!
- - Added an option to select to mute or unmute at startup. Thanks [@quyenvsp](https://github.com/quyenvsp)!
- - Fixed an issue causing a cascade of mute/unmute triggers.
+ - Move to WPF-UI, localization and light theme support. Thanks [@niels9001](https://github.com/niels9001)!
+ - Disabled by default on Windows 11, with a information box on Settings to prefer using the Windows Snipping Tool, which now supports OCR.
 
 ### Documentation
 
- - Updated the Group Policy documentation on learn.microsoft.com, removed the Group Policy documentation from the repository and linked to the published documentation on learn.microsoft.com instead.
```

Comment:
I have reworked the gpo docs on learn.ms.com again on this release and added admx, registry and intune information.